MAVProxy/tools/mavflightview.py

```diff
@@ -104,7 +104,6 @@ def create_imagefile(options, filename, latlon, ground_width, path_objs, mission
                 (mavutil.mode_mapping_rover.values(),colour_map_rover),
                 (mavutil.mode_mapping_tracker.values(),colour_map_tracker),
                 (mavutil.mode_mapping_sub.values(),colour_map_submarine),
-                (mavutil.mode_mapping_blimp.values(),colour_map_blimp),
```

Comment:
@tridge This line was giving me an error when trying to show the map, I branched from master, so this issue is also present on master. 
```
() $ MAVExplorer.py 2021-11-03\ 14-52-01.bin 
MAV> map
MAV> ERROR in command []: 'module' object has no attribute 'mode_mapping_blimp'
```

Reply:
@jmachuca77 you need to update pymavlink